core/controller/src/main/scala/whisk/core/controller/Entities.scala

```diff
@@ -65,15 +65,24 @@ protected[controller] trait ValidateRequestSize extends Directives {
   protected val fieldDescriptionForSizeError = "Request"
 }
 
+protected trait CustomHeaders extends Directives {
+  val ActivationIdHeader = "x-openwhisk-activation-id"
```

Comment:
I think @mhenke1 introduced a header with X-OW as the prefix. API gateway uses x-openwhisk. We should normalize and collocate these. Fwiw nginx also generates x-request-id header.

Reply:
@rabbah Agree.
How about using x-openwhisk?
Since x- header is a custom header specific to the application, it would be better to use the detailed one.

Reply:
I am not sure changes to use the same OpenWhisk prefix for all custom headers should be included in this PR.